src/groovy/org/pih/warehouse/api/StockMovement.groovy

```diff
@@ -240,7 +240,10 @@ class StockMovement {
         if (shipment.shipmentItems) {
             shipment.shipmentItems.each { ShipmentItem shipmentItem ->
                 StockMovementItem stockMovementItem = StockMovementItem.createFromShipmentItem(shipmentItem)
-                stockMovementItem.sortOrder = stockMovement.lineItems ? stockMovement.lineItems.size() * 100 : 0
+                if (!stockMovementItem.sortOrder) {
+                    stockMovementItem.sortOrder = stockMovement.lineItems ? stockMovement.lineItems.size() * 100 : 0
```

Comment:
@awalkowiak I just realized this might cause some weird behavior with Grails. Grails does not like gaps between sort indexes. Is there a reason we need this?